.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: Python CI
```

Comment:
This doesn't seem to be connected to the version number placement, just curious as to why this is here what it does. Not blocking merge, but interested.

Reply:
[SLACK-DISCUSSION](https://edx-internal.slack.com/archives/CM4RPEYEQ/p1637135249277900?thread_ts=1637058404.264400&cid=CM4RPEYEQ)